specs/eventing/broker.md

```diff
@@ -46,6 +46,9 @@ The attributes filter specifying a list of key-value pairs MUST be supported by
 Trigger. Events that pass the attributes filter MUST include context or
 extension attributes that match all key-value pairs exactly.
 
+If the attributes filter is not provided, Trigger MUST deliver all events to
```

Comment:
Not sure if it is a `MUST` or a `SHOULD` or something else.

Reply:
I think it's a MUST.

if not all get delivered, imo it's broken :-)